### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by EkiH**

### CODE REVIEW

File: Frends.Powershell/PowerShellHost.cs [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        }
+
+        public override SecureString ReadLineAsSecureString()
+        {
+            return new SecureString();
+        }
+
+        public override void Write(string value)
+        {
+            Trace.Write(value);
```
Comment (jvuoti) [inline Frends.Powershell/PowerShellHost.cs:27]: Could these actually write to some buffer that could then be returned in the results? So you could see any verbose output for long scripts, to see what went wrong.
But yeah, that could be a separate issue for the future.

File: Frends.Powershell/Runner.cs [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        public static PowerShellResult RunCommand(RunCommandInput input, [Browsable(false)]RunOptions options)
+        {
+            return DoAndHandleSession(options?.Session, (session) =>
+            {
+                var powershell = session.PowerShell;
+
+                var command = new Command(input.Command, isScript: false, useLocalScope: false);
+                foreach (var parameter in input.Parameters)
+                {
+                    var parameterName = parameter.Name.Trim('-', ' '); // Remove dash from start
```
Comment (jvuoti) [inline Frends.Powershell/Runner.cs:122]: why remove the dash? why would someone give it`?

File: Frends.Powershell/packages.config [hunk 3]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Management.Automation.dll" version="10.0.10586.0" targetFramework="net452" />
```
Comment (jvuoti) [inline Frends.Powershell/packages.config]: This probably is not the official package? Is there an official nuget package?

File: nuspec/Frends.Powershell.nuspec [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <id>Frends.Powershell</id>
+    <version>1.0.5.0</version>
+    <title>FRENDS Powershell tasks</title>
+    <authors>HiQ Finland</authors>
+    <owners />
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>FRENDS Powershell tasks</description>
+    <summary />
+    <dependencies>
+      <!-- <dependency id="Newtonsoft.Powershell" version="6.0.8" />
```
Comment (jvuoti) [inline nuspec/Frends.Powershell.nuspec]: strange comments ;)

File: Frends.Powershell/PowerShellHost.cs (hunk 1 above)
Comment (EkiH) [reply Frends.Powershell/PowerShellHost.cs:27]: Yeah

File: Frends.Powershell/packages.config (hunk 3 above)
Comment (EkiH) [reply Frends.Powershell/packages.config]: Removing

File: nuspec/Frends.Powershell.nuspec (hunk 4 above)
Comment (EkiH) [reply nuspec/Frends.Powershell.nuspec]: Removing

**Review by jvuoti — APPROVED** [review]

**Pull request merged**